pip.md

```diff
@@ -15,7 +15,7 @@ layout: default
 
 {% capture intro_content %}
 
-RAPIDS users can once again install RAPIDS via pip!  This is an **experimental release** supporting single GPU usage.  cuDF, dask-cuDF, cuML, cuGraph, RMM and RAFT release 22.10 pip packages are available **[right now](#install)**.  The team is excited to get these packages out into the wild and see how the RAPIDS community uses them!
+RAPIDS users can once again install RAPIDS via pip!  This is an **experimental release** supporting single GPU usage.  cuDF, dask-cuDF, cuML, cuGraph, RMM and RAFT release 22.10 pip packages are available for x86_x64 and ARM SBSA **[right now](#install)**.  The team is excited to get these packages out into the wild and see how the RAPIDS community uses them!
```

Comment:
I'd remove the last sentence and add `<i class="fa fa-angle-double-right" aria-hidden="true"></i>` to 'right now' to make a clearer call to action.

Reply:
If modifying this line anyway, we also might want to remove the 22.10 callout so we don't need to update the page again whenever 22.12 wheels are ready.